### DISCUSSION TITLE
Powerwall Capacity Meter

### DISCUSSION THREAD

**Discussion by TheNore** [jasonacox/Powerwall-Dashboard#91] [Ideas]
I was just wondering if this would be an easy add to the current dashboard - I saw it on another grafana / powerwall project.
I have been running this build for a few weeks now and love all it has to offer, but I am trying to get at a glance battery level  meter. 
any help would be appreciated.

![image](https://user-images.githubusercontent.com/16988567/193472927-7830d0a0-9096-46f8-95f8-979d1eedc31d.png)

**Comment by jasonacox** [comment]
I like it! Where did you see this? We could take a peak at the grafana panel they are using and do something similar.  

I like the "Peak Output" part too but is that Solar generation peak or Powerwall peak output?

**Reply by TheNore** [reply]
Thanks! Here's the link I found it at.
The peak would be for powerwall unit I'd think, but that's above the rated amount of the unit...

https://github.com/rhodesman/teslaPowerDash

**Reply by TheNore** [reply]
Is there any plan for this to be integrated?

**Reply by jasonacox** [reply]
TODO: Figure out a way to have a marketplace of panels or dashboards for the community to use.   

I have this meter on my list but welcome any help.  I think the approach could start as simple as adding a README.md index to /dashboards folder and create some folder structure there to represent the different dashboards and add-ons for panels like this.  This particular meter would require some refactoring. I haven't had time to dig in too deep yet.  Please feel free to take a look and if you can get it to work, post a PR or details so we can pull it into this project.
